components/ui/Graph/Graph.js

```diff
@@ -0,0 +1,90 @@
+'use client';
+import { Line } from 'react-chartjs-2';
+
+import {
+  CategoryScale,
+  Chart as ChartJS,
+  Filler,
+  Legend,
+  LineElement, // x axis
+  LinearScale, // y axis
+  PointElement,
+  Tooltip,
+} from 'chart.js';
+
+ChartJS.register(
+  LineElement,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  Legend,
+  Tooltip,
+  Filler
+);
+
+function Graph({ graphData, graphConfig, color }) {
+  const data = {
+    labels: graphData.map((data) => data.month),
+    datasets: [
+      {
+        label: 'Revenue',
+        data: graphData.map((data) => data.sales),
+        borderColor: color,
+        borderWidth: 3,
+        pointBorderColor: color,
+        pointBorderWidth: 3,
+        pointRadius: 0,
+        tension: 0.5,
+        fill: true,
+        // backgroundColor: (context) => {
```

Comment:
If these commented codes are no longer required then we will not put it into PR, it is better to remove it.

Reply:
Okay brother!